substrate/frame/nomination-pools/src/migration.rs

```diff
@@ -59,22 +59,7 @@ pub mod versioned {
 pub mod v8 {
 	use super::*;
 
-	#[derive(Decode)]
-	pub struct OldCommission<T: Config> {
-		pub current: Option<(Perbill, T::AccountId)>,
-		pub max: Option<Perbill>,
-		pub change_rate: Option<CommissionChangeRate<BlockNumberFor<T>>>,
-		pub throttle_from: Option<BlockNumberFor<T>>,
-	}
-
-	#[derive(Decode)]
-	pub struct OldBondedPoolInner<T: Config> {
-		pub commission: OldCommission<T>,
-		pub member_counter: u32,
-		pub points: BalanceOf<T>,
-		pub roles: PoolRoles<T::AccountId>,
-		pub state: PoolState,
-	}
+	use super::v7::{BondedPoolInner as OldBondedPoolInner, Commission as OldCommission};
```

Comment:
```suggestion
	use super::v7::BondedPoolInner as OldBondedPoolInner;
```

`OldCommission` is not used here